mlflow/tensorflow/_autolog.py

```diff
@@ -52,11 +59,162 @@ def on_epoch_end(self, epoch, logs=None):
         if epoch % self.log_every_n_steps == 0:
             self.metrics_logger.record_metrics(logs, epoch)
 
-    def on_train_end(self, logs=None):  # pylint: disable=unused-argument
-        if self.log_models:
-            registered_model_name = get_autologging_config(
-                mlflow.tensorflow.FLAVOR_NAME, "registered_model_name", None
-            )
-            mlflow.keras.log_model(
-                self.model, artifact_path="model", registered_model_name=registered_model_name
-            )
+
+def _extract_input_example_from_tensor_or_ndarray(
+    input_features: Union[tensorflow.Tensor, np.ndarray]
+) -> np.ndarray:
+    """
+    Extracts first `INPUT_EXAMPLE_SAMPLE_ROWS` from the next_input, which can either be of
+    numpy array or tensor type.
+
+    :param input_features: an input of type `np.ndarray` or `tensorflow.Tensor`
+    :return: a slice (of limit `INPUT_EXAMPLE_SAMPLE_ROWS`)  of the input of type `np.ndarray`
```

Comment:
Can we update this to also say that we return `None` if the type of `input_features` is unsupported?